app/views/categories/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<nav>
+  <div><h2>Categories</h2></div>
+  <%= button_to 'Sign Out', destroy_user_session_path, method: :delete %>
+</nav>
+<section class="main">
+  <% @categories.each do |category| %>
+    <section class="category">
+      <div>
+        <img src="<%= category.icon %>">
+        <%= link_to category_procurements_path(category) do %>
+          <%= category.name %>
+        <% end %>
```

Comment:
![image](https://github.com/Wineshuga/buddy_budget/assets/93039359/15387534-db24-4e11-a079-741b61db18ce)

- [ ] It would be nice if you fix the images because I have added two icons on two categories but none of them is showing 😨 Kindly fix them to make your design look professional by fixing the images 👍